pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -41,13 +41,10 @@
       </Dependency>
     </ItemGroup>
 
-    <!-- Use NETStandard.Library version suffix for version suffix of Microsoft.NETCore.App -->
     <PropertyGroup>
-      <NETStandardBuildNumber Condition="'%(Dependency.Identity)' == 'NETSTandard.Library'">$([System.Text.RegularExpressions.Regex]::Match('%(Dependency.Version)', -.*))</NETStandardBuildNumber>
-      <VersionSuffix Condition="'$(NETStandardBuildNumber)' != ''">$(NETStandardBuildNumber)</VersionSuffix>
-      <VersionSuffix Condition="'$(CLIBuildVersion)' != ''">$(VersionSuffix)-$(CLIBuildVersion)</VersionSuffix>
+    <!-- For RC2  we need to start with -rc2-3 so that it is higher then the last published -rc2-23931 -->
+    <VersionSuffix>-rc2-3$(CLIBuildVersion)</VersionSuffix>
```

Comment:
For now I'm to keep it here. The more I dug in on this the more I think we need to clean-up our version passing between the CLI build system and the BuildTools system, and I will do that as a secondary change later.